src/components/MainComponent.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-  <div class="outer-container">
+  <div class="outer-container" style="position:relative; overflow-y: hidden">
+    <Circles />
     <Navigation @selected="setActive" />
-    <div class="main-content">
+    <div class="main-content" style="position: relative">
```

Comment:
use bootstrap for this
```suggestion
    <div class="main-content position-relative">
```